packages/connect/src/device/Device.ts

```diff
@@ -544,8 +544,13 @@ export class Device extends TypedEmitter<DeviceEvents> {
         this.features = feat;
         this.featuresNeedsReload = false;
 
-        // Firmware type is unknown in bootloader mode
-        if (this.getMode() !== 'bootloader') {
+        // Vendor headers have been changed in 2.6.3.
+        if (feat.fw_vendor === 'Trezor Bitcoin-only') {
```

Comment:
I'd be inclined to define those possible headers into constants somewhere, but as long as it's a single use, it's probably OK to have strings.